pkg/kafka/configkafka/config.go

```diff
@@ -53,6 +53,13 @@ type ClientConfig struct {
 	// replica selection when supported by the brokers. This maps to Kafka's
 	// standard "client.rack" setting. By default, this is empty.
 	RackID string `mapstructure:"rack_id"`
+
+	// UseLeaderEpoch forces the collector to clear the leader epoch from
+	// all fetched offsets when using the franz-go consumer. This effectively
+	// disables KIP-320 (truncation detection via leader epoch) without
+	// downgrading protocol versions or patching franz-go.
+	// Default: false (leader epoch is used when supported by the broker).
+	UseLeaderEpoch bool `mapstructure:"use_leader_epoch"`
```

Comment:
```suggestion
	//
	// NOTE: this is experimental and may be removed in a future release.
	UseLeaderEpoch bool `mapstructure:"use_leader_epoch"`
```

On second thoughts, let's mention it here too. Otherwise anyone directly depending on the config struct may get a surprise.